release-notes-3.3.7.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
 
 #### New Features
+- US1009117: Invalid message queue configuration added.
```

Comment:
You're introducing a new feature here, so the version number should be moving to `3.4.0-SNAPSHOT`.